fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/TypeRdfContext.java

```diff
@@ -66,10 +66,12 @@ private void concatRdfTypes() throws RepositoryException {
         final NodeType primaryNodeType = resource().getNode().getPrimaryNodeType();
         nodeTypesB.add(primaryNodeType);
```

Comment:
If `primaryNodeType` is null, an NPE will be thrown on line 67.
I would suggest checking if `primaryNodeType` is null between lines 66 and 67.